Artifacts/linux-bitcore/install_bitcore.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -x
+echo "Initializing Bitcore installation"
+
+#############
+# Parameters
+#############
+AZUREUSER=$1
+HOME="/home/$AZUREUSER"
+echo "User: $AZUREUSER"
+echo "User home dir: $HOME"
+
+yum -y install epel-release
+yum -y install npm git curl which xz tar findutils
```

Comment:
Not all Linux distros have yum. In other artifacts we've added conditional support for apt-get or yum. Here's an example: https://github.com/Azure/azure-devtestlab/blob/master/Artifacts/linux-install-mongodb/linux_install_mongodb_v1.sh